routes/api.php

```diff
@@ -17,6 +18,8 @@
 
 Route::post('/login', [AuthController::class, 'login']);
 
+Route::get('tasks', [TaskController::class, 'show']);
```

Comment:
add "/" to the route and change the method to "index" instead of "show".
Overall is perfect. Great job Mark